### DISCUSSION TITLE
Zigbee2mqtt doesn't start after reboot

### DISCUSSION THREAD

**Discussion by pmol200** [Koenkk/zigbee2mqtt#10451] [Devices]
My Pi was perfectly working with Home Assistant, Zigbee2mqtt, mqtt and more. but after a reboot, zigbee2mqtt doesn't work anymore.
The log is listed below.

I use a CC2531 coordinator, and my firmware is quite old, from 2018. Of course I'd like to fix the issue with the current configuration, but would it be wise to upgrade the firmware? I have no equipment to flash the firmware, but I will buy if required.

`Zigbee2MQTT:info  2021-12-30 17:50:35: Starting Zigbee2MQTT version 1.22.1 (commit #c2b5229)
Zigbee2MQTT:info  2021-12-30 17:50:35: Starting zigbee-herdsman (0.13.176)
Zigbee2MQTT:error 2021-12-30 17:50:56: Error while starting zigbee-herdsman
Zigbee2MQTT:error 2021-12-30 17:50:56: Failed to start zigbee
Zigbee2MQTT:error 2021-12-30 17:50:56: Check https://www.zigbee2mqtt.io/guide/installation/20_zigbee2mqtt-fails-to-start.html for possible solutions
Zigbee2MQTT:error 2021-12-30 17:50:56: Exiting...
Zigbee2MQTT:error 2021-12-30 17:50:56: Error: SRSP - SYS - getExtAddr after 6000ms
    at Timeout._onTimeout (/app/node_modules/zigbee-herdsman/src/utils/waitress.ts:64:35)
    at listOnTimeout (internal/timers.js:557:17)
    at processTimers (internal/timers.js:500:7)`

**Comment by Koenkk** [comment]
> Of course I'd like to fix the issue with the current configuration, but would it be wise to upgrade the firmware? I have no equipment to flash the firmware, but I will buy if required.

2018 firmwares are indeed very old (beginning of z2m 😄 ), I would save the hassle of buying the flashing hardware, this will be around 10$ while the much more powerful sonoff 3.0 zigbee plus dongle is also around that price.

**Reply by pmol200** [reply]
Thanks for the advise, I'll look for the sonoff 3.0 device.
But as my configuration was working, would it be able to get it running again, till I have the new device? Is this a known issue with the stability of the CC2531 stick?

**Reply by Koenkk** [reply]
yes this is a known issue of the cc2531 stick, the memory can get corrupted, this is the case if replugging the stick doesn't fix the issue. Once it is corrupted only reflashing helps
